core/read.c

```diff
@@ -2690,6 +2690,68 @@ static int isom_read_skip_box_extra_bytes( lsmash_file_t *file, isom_box_t *box,
     return 0;
 }
 
+#define READ_STRING( box_name, field )                                            \
```

Comment:
I thought for sure that L-SMASH had a C string reading function, but it seems it does not, since all the other boxes with strings it supports have them at the end of the box (and thus their length is calculable via box size).

Might be worth making this macro into a function or something in this file.

Reply:
Ok, I can make it a proper function, taking pointers to the fields where it stores the pointer and length.